packages/storage/src/client_storage/fs.rs

```diff
@@ -9,6 +9,38 @@ use tokio::sync::watch::{Receiver, channel};
 
 use crate::{StorageBacking, StorageSubscriber, serde_to_string, try_serde_from_string};
 
+
+#[doc(hidden)]
+#[cfg(target_os = "android")]
+pub fn data_directory() -> std::path::PathBuf {
+    use jni::objects::{JObject, JString};
```

Comment:
Maybe this feature should be added after https://github.com/DioxusLabs/dioxus/pull/4842, when platform-bridge is merged? So that we don't need to rely on dioxus mobile renderer here.

Reply:
We can revisit this once that PR is stabilized without any breaking changes, but it doesn't need to block android support for storage